### ISSUE TITLE
About loss function

### ISSUE THREAD

**Issue by zyh3826** [princeton-nlp/SimCSE#97]
Why is the code using cross-entropy(at model.py line 201) instead of the loss function mentioned in the paper? Do I miss something?
the loss function in the paper:
![image](https://user-images.githubusercontent.com/31238754/134612803-15d1c8da-6976-4a9b-b70a-e505a61e92b9.png)

**Comment by gaotianyu1350** [comment]
Hi,

Cross entropy function of Pytorch produces the same thing as this function.

**Comment by zyh3826** [comment]
> Hi,
> 
> Cross entropy function of Pytorch produces the same thing as this equation.

Thanks for your reply, did you set the temperature t to 1？

**Comment by gaotianyu1350** [comment]
We set the temperature as 0.05 (mentioned in the paper).

**Comment by zyh3826** [comment]
> We set the temperature as 0.05 (mentioned in the paper).

got it! Thanks
